validate.js

```diff
@@ -0,0 +1,57 @@
+function showMessage(input, message, type) {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+  input.className = type ? 'success' : 'error';
+  return type;
+}
+
+function showError(input, message) {
+  return showMessage(input, message, false);
+}
+
+function showSuccess(input) {
+  return showMessage(input, 'data added', true);
+}
+
+function hasValue(input, message) {
+  if (input.value.trim() === '') {
+    return showError(input, message);
+  }
+  return showSuccess(input);
+}
+
+function validateEmail(input, requiredMsg, invalidMsg) {
+  if (!hasValue(input, requiredMsg)) {
+    return false;
+  }
+
+  const emailRegex = '';
+  // /^([a-z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|
+  // (([a-z0-9\-]+\.)+))([a-z]{2,4}|[0-9]{1,3})(\]?)$/;
```

Comment:
Hi @GordonOtieno ✋

- [ ] [Optional] You may please remove commented code from your validate.js file. Leave commented code in your project is not regarded a good practice.